owrrm/src/test/java/com/facebook/openwifi/rrm/optimizers/tpc/LocationBasedOptimalTPCTest.java

```diff
@@ -141,6 +144,10 @@ void testLocationBasedOptimalTPCSuccessCase() throws Exception {
 						DEFAULT_CHANNEL_5G,
 						DEFAULT_CHANNEL_WIDTH,
 						DEFAULT_TX_POWER,
+						dummyBssid,
```

Comment:
this State the DeviceCapability below can be created once and used twice, if you want to save some lines of code -- might make it more readable (your call)